x/gov/keeper/config.go

```diff
@@ -26,6 +26,8 @@ type Config struct {
 	MaxMetadataLen uint64
 	// MaxSummaryLen defines the amount of characters that can be used for proposal summary
 	MaxSummaryLen uint64
+	// MaxVoteOptionsLen defines the maximum number of vote options a proposal can have
```

Comment:
Can we explain that it is for the weighted voting message only and isn't related to actual proposal options? It could get confusing otherwise. Maybe an example could help.

Can add here too that 0 means disabled?

Reply:
Done